src/lib/sw.js

```diff
@@ -15,31 +14,31 @@ import {matchSameOriginRegExp} from './utils/sw-match.js';
 
 const cacheNames = {webdevCore: 'webdev-core', ...workboxCacheNames};
 
+// This import defines self['_manifest'], used below.
```

Comment:
Suggest expanding on this comment a bit to explain that this is the workbox precache manifest and we inject it ourselves instead of using the self.__WEB_MANIFEST approach in the workbox docs.